jcphuff.c

```diff
@@ -680,7 +680,7 @@ encode_mcu_AC_first(j_compress_ptr cinfo, JBLOCKROW *MCU_data)
       emit_restart(entropy, entropy->next_restart_num);
 
 #ifdef WITH_SIMD
-  cvalue = values = (JCOEF *)PAD((size_t)values_unaligned, 16);
+  cvalue = values = (JCOEF *)PAD((uintptr_t)values_unaligned, 16);
```

Comment:
Th `PAD()` calls in this file could be avoided if we were able to use C11 `alignas()`, but I assume that older compilers need to be supported?

Reply:
Correct.  C89 is the best we can do at the moment, which is at least an improvement over libjpeg (which originally had to support non-ANSI C and MS-DOS.)